config/settings_manager.py

```
@@ -391,12 +402,69 @@ def update_risk_config(self, **kwargs) -> None:
             if hasattr(self.config.risk, key):
                 setattr(self.config.risk, key, value)
         self.save_config()
-    
+
     def update_gui_config(self, **kwargs) -> None:
         """GUI 설정 업데이트"""
         self.config.gui.update(kwargs)
         self.save_config()
-    
+
+    def get_trading_config(self, exchange: str = None) -> TradingConfig:
```

Ruff (0.14.1)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)